public/app/core/components/Permissions/DisabledPermissionsListItem.tsx

```diff
@@ -13,7 +13,7 @@ export default class DisabledPermissionListItem extends Component<IProps, any> {
     return (
       <tr className="gf-form-disabled">
         <td style={{ width: '1%' }}>
-          <i style={{ width: '25px' }} className="gicon gicon-shield" />
+          <i style={{ width: '25px', fontSize: '1.5rem' }} className="gicon gicon-shield" />
```

Comment:
Would be nice with a specific gicon-class name for this in scss if not already exists. Seems like this style is needed as well in PermissionListItem.tsx (when there are no avatar).